### PR TITLE
COMMERCE-10421

### PR DESCRIPTION
**Pull request by Brunno0liveira**
https://liferay.atlassian.net/browse/COMMERCE-10421

### CODE REVIEW

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by saramart81** [comment]
ci:test:relevant

**Comment by saramart81** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceCurrencies.testcase [hunk 1]
```diff
@@ -60,6 +60,42 @@ definition {
...
+	@description = "This is a test for COMMERCE-10421. As admin, I can search a currency."
+	@priority = 5
+	test CanSearchCurrency {
+		property portal.acceptance = "true";
+
+		task ("Given a admin goes to the currency admin page") {
+			CommerceNavigator.goToCurrenciesAdmin();
+		}
+
+		task ("And teh user search for a currency") {
```
Comment (saramart81) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceCurrencies.testcase:72]: "And the user search for a currency"

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
ci:test:stable

**Comment by Brunno0liveira** [comment]
ci:test:stable

**Comment by victoroscuro** [comment]
ci:test:stable

**Comment by 22LucaGiannotta22** [comment]
ci:test:stable

**Comment by victoroscuro** [comment]
ci:test:stable

**Comment by jaydawu** [comment]
ci:test:stable

**Comment by jaydawu** [comment]
ci:test:stable

**Comment by jaydawu** [comment]
Hi @victoroscuro, the stable job is passing now, please help force-forward it, thanks!

**Comment by victoroscuro** [comment]
ci:forward:force

**Pull request closed without merging**